docs/overview.md

```diff
@@ -11,11 +11,12 @@ Benefits include:
 
 ## Modes
 
-Psoxy can be deployed/used in 4 different modes:
+Psoxy can be deployed/used in 4 different modes, to support various data sources:
 
 - **API** - psoxy sits in front of a data source API. Any call that would normally be sent to the data source API is instead sent to psoxy, which parses the request, validates it / applies ACL, and adds authentication before forwarding to the host API. After the host API response, psoxy sanitizes the response as defined by its roles before returning the response to the caller. This is an _http triggered_ flow.
```

Comment:
Minor note: Throughout the docs, we use both “psoxy” and “Psoxy.” Shouldn’t we always capitalize it, since it’s the name of our product?